packages/pwa-kit-mcp/src/utils/telemetry.js

```diff
@@ -156,4 +147,17 @@ export class Telemetry {
         this.started = false
         this.reporter?.stop()
     }
+
+    async createMcpTelemetryReporter() {
```

Comment:
I understand we have both the key and the o11y endpoint, and this is where we would modify based on our approved approach, should we add a Todo or comment here?